lib/bunny_mock.rb

```diff
@@ -157,13 +166,14 @@ def method_missing(method, *args)
   end # class Queue
 
   class Exchange
-    attr_accessor :channel, :type, :name, :attrs, :queues
-    def initialize(channel, type, name, attrs = {})
+    attr_accessor :channel, :type, :name, :options, :queues
+
+    def initialize(channel, type, name, opts = {})
       self.channel = channel
-      self.type = type
-      self.name   = name
-      self.attrs  = attrs.dup
-      self.queues = []
+      self.type    = type
+      self.name    = name
+      self.options = opts.dup
```

Comment:
this one is tricky because Bunny has `opts` as attr accessible and `@options` as the variable they set in the initializer. I believe that's a problem on their side since the other classes all use `:options` and not `:opts`